src/app/utils/codemirror.js

```diff
@@ -70,6 +71,8 @@ export const getCodeMirror = (el, doc) => {
     .cm-s-oceanic div.cm-line-error.CodeMirror-linebackground { animation: ${fadeInAnimation} 0.3s; background-color: #561011; }
   `;
 
+  emmet(CodeMirror);
```

Comment:
I think this could be moved outside of the function (more to top), because this only has to be run once in application lifetime.